vembrane/cli.py

```diff
@@ -1,10 +1,14 @@
 import argparse
 
 from .modules import filter, table
+from . import __version__
 
 
 def main():
     parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--version", "-v", action="version", version=f"%(prog)s {__version__}"
```

Comment:
```suggestion
        "--version", action="version", version=f"%(prog)s {__version__}"
```
`-v` is also commonly used as a short option for `--verbose`. I would leave that one out.

Reply:
Ah, yes, I forgot.